.github/workflows/ghcr-image-build-and-publish.yml

```diff
@@ -21,9 +21,8 @@ env:
   # github.repository as <account>/<repo>
   IMAGE_NAME: ${{ github.repository }}
 
-
 jobs:
-  build:
+  publish:
```

Comment:
Why? This doesn’t always publish the image

Reply:
The idea was to help distinguish it better from the other `build` job when displayed in the checks.
Either way is fine with me anyhow.

Reply:
Reverted to build.